src/AmazonPHP/SellingPartner/CredentialsHandler.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Plenty\AmazonPHP\SellingPartner;
+
+
+use Plenty\AmazonPHP\SellingPartner\STSClient\Credentials;
+
+class CredentialsHandler {
+    /** @var Credentials  */
+    private Credentials $credentials;
+
+    /** @var array  */
+    private array $secrets;
+
+    /** @var STSClient  */
+    private STSClient $sts;
+
+    /** @var string  */
+    private string $roleArn;
+
+    public function __construct(STSClient $sts, array $secrets, string $roleArn)
```

Comment:
Let us please split the secrets param into client ID and client secret

Reply:
Done